tools/content-generator/src/test/java/org/projectnessie/tools/contentgenerator/AbstractContentGeneratorTest.java

```diff
@@ -75,20 +75,20 @@ protected static final class ProcessResult {
     private final int exitCode;
     private final String stdOut;
 
-    protected ProcessResult(int exitCode, String stdOut) {
+    ProcessResult(int exitCode, String stdOut) {
       this.exitCode = exitCode;
       this.stdOut = stdOut;
     }
 
-    protected int getExitCode() {
+    int getExitCode() {
       return exitCode;
     }
 
-    protected String getStdOut() {
+    private String getStdOut() {
```

Comment:
not sure if this should be private, since its part of the "interface"... it just happens to not be called atm